scopes/ui-foundation/use-box/menu/menu.tsx

```diff
@@ -33,7 +34,7 @@ export function Menu({ methods, componentName, ...rest }: MenuProps) {
   const { Component } = OrderedMethods[activeTab] || {};
 
   return (
-    <div {...rest}>
+    <div {...rest} className={classNames(rest.className, styles.container)}>
```

Comment:
pass also the className from the props.
`<div className={classNames(rest.className, styles.container, className)} {...rest}>`

Reply:
`rest.className` is from props